types/api/order.d.ts

```diff
@@ -0,0 +1,187 @@
+import { Part } from "../base";
+
+export type OrderState = {
+    id: number;
+    enabled: boolean;
+    description?: string;
+    created: number;
+    modified: number;
+    meta: Record<string, unknown>;
+    brand_t?: string;
+    retailer_t?: string;
+    vendor_t?: string;
+    name: string;
+    justification?: string;
+    timestamp: number;
+    email: string;
+}
+
+export type Attachment = {
```

Comment:
Since this is its own separate model then move it to its own file.

As a rule of thumb, the files for types should be 1-1 with the files of the models.